awx/main/models/base.py

```diff
@@ -61,7 +60,16 @@
     (PERM_INVENTORY_CHECK, _('Check')),
 ]
 
-CLOUD_INVENTORY_SOURCES = list(CLOUD_PROVIDERS) + ['scm']
+
+def compute_cloud_inventory_sources() -> list[str]:
+    """Return a list of cloud provider plugin names
+    available plus source control management.
+
+    :returns: List of plugin cloud names plus sou.
+    :rtype: list[str]
+    """
+    return discover_available_cloud_provider_plugin_names() + ['scm']
+
```

Comment:
Perhaps, this should also move to `awx/main/utils/plugins.py` since it's not a model?
```suggestion
```